linkage_dag.py

```diff
@@ -341,19 +342,55 @@
         ),
     ]
 
-    # Start up godzilla of article linkage, create the merged ids
+    gce_instance_create = ComputeEngineInsertInstanceOperator(
+        task_id=f"create_{gce_resource_id}",
+        project_id=PROJECT_ID,
+        zone=GCP_ZONE,
+        body={
+            "name": gce_resource_id,
+            "machine_type": f"zones/{GCP_ZONE}/machineTypes/m1-megamem-96",
+            "disks": [
+                {
+                    "boot": True,
+                    "auto_delete": True,
+                    "initialize_params": {
+                        "disk_size_gb": "500",
+                        "disk_type": f"zones/{GCP_ZONE}/diskTypes/pd-balanced",
+                        "source_image": "projects/ubuntu-os-cloud/global/images/ubuntu-2204-jammy-v20240927",
+                    },
+                }
+            ],
+            "network_interfaces": [
+                {
+                    "access_configs": [
+                        {"name": "External NAT", "network_tier": "PREMIUM"}
+                    ],
+                    "stack_type": "IPV4_ONLY",
+                    "subnetwork": "regions/us-east1/subnetworks/default",
+                }
+            ],
+            "service_accounts": [
+                {
+                    "email": "dataloader@gcp-cset-projects.iam.gserviceaccount.com",
+                    "scopes": [
+                        "https://www.googleapis.com/auth/devstorage.full_control",
+                        "https://www.googleapis.com/auth/cloud-platform",
+                    ],
+                }
+            ],
+        },
+    )
+
     gce_instance_start = ComputeEngineStartInstanceOperator(
-        project_id=project_id,
-        zone=gce_zone,
+        task_id=f"start-{gce_resource_id}",
+        project_id=PROJECT_ID,
+        zone=GCP_ZONE,
         resource_id=gce_resource_id,
-        task_id="start-" + gce_resource_id,
     )
 
     prep_environment_script_sequence = [
         f"/snap/bin/gsutil cp gs://{bucket}/{gcs_folder}/vm_scripts/*.sh .",
-        "cd /mnt/disks/data",
-        "rm -rf run",
-        "mkdir run",
+        "rm -rf run; mkdir run",
```

Comment:
If this is a new disk every time, do we still need to rm -rf or is this going to fail every time?

Reply:
This will clear out the directory if the task is retried